src/__tests__/calculatorModel.test.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  calculate,
+  updateAccumulator,
+  updateOperand,
+  updateOperation,
+  ICalculator,
+} from 'components/Calculator/CalculatorModel';
+import { Digit, OPERATION } from 'constants/calculator';
+
+const initialState: ICalculator = {
+  operation: null,
+  leftOperand: 0,
+  rightOperand: null,
+  accumulator: '0',
+};
+
+describe('Calculator Model', () => {
+  test('연산자를 저장할 수 있어야 한다.', () => {
+    let calculator = { ...initialState };
```

Comment:
```suggestion
describe('Calculator Model', () => {
  beforeEach(() => {
      let calculator = { ...initialState };
  });
  
  test('연산자를 저장할 수 있어야 한다.', () => {
```
이렇게 beforeEach에서 각각의 테스트 시작 전에 실행할 코드를 정의할 수 있답니다!

Reply:
아 ~ 이런 식으로 쓸 수가 있었군요. 감사합니다! 👍